examples/audio/ctc_asr.py

```diff
@@ -337,11 +324,38 @@ def build_model(input_dim, output_dim, rnn_layers=5, rnn_units=128):
 """
 
 
+# Reference: https://github.com/keras-team/keras/blob/ec67b760ba25e1ccc392d288f7d8c6e9e153eea2/keras/legacy/backend.py#L715-L739
+
+
+def ctc_decode(y_pred, input_length, greedy=True, beam_width=100, top_paths=1):
+    input_shape = tf.shape(y_pred)
+    num_samples, num_steps = input_shape[0], input_shape[1]
+    y_pred = tf.math.log(tf.transpose(y_pred, perm=[1, 0, 2]) + keras.backend.epsilon())
+    input_length = tf.cast(input_length, tf.int32)
+
+    if greedy:
+        (decoded, log_prob) = tf.nn.ctc_greedy_decoder(
```

Comment:
So, we're going to have to use TF for this and `ctc_beam_search_decoder` I guess, unless we implement them as new backend ops.

Reply:
Again, thanks for the feedback 👍
I created an [issue](https://github.com/keras-team/keras/issues/19227) to address this.
Please let me know if I should change the description or add/remove details.
Thanks!